examples/gocean/eg5/readonly/README.md

```diff
@@ -1,4 +1,4 @@
-# PSyclone GOcean Example 7
+# PSyclone GOcean PSyData Example 3 - ReadOnly-Verification
```

Comment:
As with the others, I wouldn't number these since the directories (under eg5) aren't numbered: "...PSyData  Read-Only Verification Example"